discovery-infra/test_infra/controllers/node_controllers/vsphere_controller.py

```diff
@@ -0,0 +1,154 @@
+import os
+from builtins import list
+from typing import Tuple, List, Callable, Any
+
+import libvirt
+
+from test_infra.controllers.node_controllers.disk import Disk
+from test_infra.controllers.node_controllers.node import Node
+from test_infra.controllers.node_controllers.node_controller import NodeController
+from test_infra.helper_classes.config import BaseClusterConfig
+from test_infra.helper_classes.config.vsphere_config import VSphereControllerConfig
+from test_infra.tools import terraform_utils
+from test_infra.utils.terraform_util import TerraformControllerUtil
+
+
+class VSphereController(NodeController):
+
+    def __init__(self, config: VSphereControllerConfig, cluster_config: BaseClusterConfig):
+        super().__init__(config, cluster_config)
```

Comment:
`NodeController` has no `__init__` function that gets `VSphereControllerConfig` and   `BaseClusterConfig`

Reply:
It has changed before this PR, It's ok